R/responsibilities.R

```diff
@@ -0,0 +1,318 @@
+helper_touchstones <- function(modelling_group_id, location = NULL) {
+  assert_scalar_character(modelling_group_id)
+  path <- sprintf("/modelling-groups/%s/responsibilities/", modelling_group_id)
+  montagu_api_GET(location, path)
+}
+
+##' This may get a name change pending splitting apart of various
+##' montagu components.
+##' @title Retrieve touchstones a modelling group is responsible for.
+##' @param location The montagu server to connect to.
+##' @param modelling_group_id Id of the modelling group.
+##' @return Data frame of touchstone name, description and comment.
+##' @export
+montagu_touchstones <- function(modelling_group_id, location = NULL) {
+  res <- helper_touchstones(modelling_group_id, location)
+  data_frame(
+    name = vcapply(res, "[[", "id"),
+    description = vcapply(res, "[[", "description"),
+    comment = vcapply(res, "[[", "comment"))
+}
+
+##' This may get a name change pending splitting apart of various
+##' montagu components.
+##' @title Retrieve list of all versions of a given touchstone.
+##' @param touchstone_name Name of the touchstone (ie, no version suffix)
+##' @inheritParams montagu_touchstones
+##' @return Data frame of touchstine id, name, version, description and status
+##' @export
+montagu_touchstone_versions <- function(modelling_group_id,
+                                        touchstone_name, location = NULL) {
+  assert_scalar_character(touchstone_name)
+  res <- helper_touchstones(modelling_group_id, location)
+
+  if (length(res)==0) {
```

Comment:
style wise, could you put spaces around `==` please

Reply:
Done - always a couple sneak through....